filament/src/materials/blur.mat

```diff
@@ -0,0 +1,108 @@
+material {
+    name : blur,
+    parameters : [
+        {
+            type : sampler2d,
+            name : ssao,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : depth,
+            precision: medium
+        },
+        {
+            type : float4,
+            name : resolution
+        },
+        {
+            type : int2,
+            name : axis
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    vertexDomain : device,
+    depthWrite : false,
+    depthCulling : true,
+    shadingModel : unlit,
+    variantFilter : [ skinning, shadowReceiver ],
+    culling: none
+}
+
+vertex {
+    void materialVertex(inout MaterialVertexInputs material) {
+        // far-plane in view space
+        vec4 position = getPosition(); // clip-space
+        position.z = 1.0; // far plane
+        material.vertex.xy = (position.xy * 0.5 + 0.5);
+        material.vertex.zw = position.xy;
+    }
+}
+
+fragment {
+    // z-distance (in m) that constitute an edge for bilateral filtering
+    #define EDGE_DISTANCE   0.1
+
+    const int kGaussianCount = 5;
+    const int kRadius = kGaussianCount - 1;
+    const float kGaussianSamples[] = float[](
+        0.239365, 0.199935, 0.116512, 0.0473701, 0.0134367
+    );
+    const float kGaussianWeightSum = 0.993872;
```

Comment:
Is `kGaussianWeightSum` used anywhere?

Reply:
It's not anymore because it now depends on the bilateral weights which are different at each pixel. But it's part of the "gaussian" filter data, so I left it there.